pages.id/common/rbenv.md

```diff
@@ -0,0 +1,36 @@
+# rbenv
+
+> Alat untuk menginstal versi Ruby dan mengatur lingkungan aplikasi dengan mudah.
+> Informasi lebih lanjut: <https://github.com/rbenv/rbenv>.
+
+- Instal suatu versi Ruby:
+
+`rbenv install {{versi}}`
+
+- Menampilkan daftar versi-versi stabil terbaru dari Ruby:
+
+`rbenv install --list`
+
+- Menampilkan daftar versi-versi Ruby yang terinstal:
```

Comment:
I don't speak the language, but I'm not sure about the verbs. The verbs should be in imperative mood.